Mathlib/Init/Data/List/Lemmas.lean

```diff
@@ -72,20 +72,25 @@ alias ⟨eq_or_mem_of_mem_cons, _⟩ := mem_cons
 #align list.mem_append_left List.mem_append_left
 #align list.mem_append_right List.mem_append_right
 
-theorem not_bex_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x := fun ⟨_, hx, _⟩ => List.not_mem_nil _ hx
-#align list.not_bex_nil List.not_bex_nil
+theorem not_exists_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
+  fun ⟨_, hx, _⟩ => List.not_mem_nil _ hx
+#align list.not_bex_nil List.not_exists_nil
 
 #align list.ball_nil List.forall_mem_nil
 
-theorem bex_cons (p : α → Prop) (a : α) (l : List α) : (∃ x ∈ a :: l, p x) ↔ p a ∨ ∃ x ∈ l, p x :=
+theorem exists_cons (p : α → Prop) (a : α) (l : List α) :
```

Comment:
```suggestion
theorem exists_mem_cons (p : α → Prop) (a : α) (l : List α) :
```

Reply:
This one already existed, i.e. this lemmas was a duplicate. Changed the align and alias accordingly and deleted this lemma.